src/EnergyPlus/HVACManager.cc

```diff
@@ -491,6 +489,9 @@ void ManageHVAC(EnergyPlusData &state)
         FirstTimeStepSysFlag = false;
     } // system time step  loop (loops once if no downstepping)
 
+    state.dataHeatBalFanSys->ZTAVComf = state.dataHeatBalFanSys->ZTAV;
+    state.dataHeatBalFanSys->ZoneAirHumRatAvgComf = state.dataHeatBalFanSys->ZoneAirHumRatAvg;
+
```

Comment:
A small change to make such an impact.  But it seems reasonable, and analysis of the diffs shows better results.  This is fine.